index.html

```diff
@@ -8,6 +8,7 @@
        <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
 
        <title>PixelVibe</title>
+       <link rel="favicon" href="assets/20210325_164428 pixelvibe.svg" type="image/svg" >
```

Comment:
There will be many such `<link>` added to the index.html and you don't have to add your SVG, we will use ico files. please wait until #91 is merged and read the StackOverflow post again. Let me o=know if you don't get how to do it.